Debian #339790; GNOME #322137; patch from CVS to fix new auto VT allocation
method.

Comment from Josselin: No idea what this patch was introduced for. The 
references have nothing to do with it and concern a bug fixed long ago.
It seems to patch the build system for only half of the binaries, and I 
don't see the point.
Index: gdm-2.18.1/gui/Makefile.am
===================================================================
--- gdm-2.18.1.orig/gui/Makefile.am	2007-04-09 07:15:11.000000000 +0200
+++ gdm-2.18.1/gui/Makefile.am	2007-05-02 11:07:23.000000000 +0200
@@ -119,10 +119,7 @@
 	$(PANGO_LIBS)		\
 	$(GDKPIXBUF_LIBS)	\
 	$(top_builddir)/vicious-extensions/libviciousui-minimal.a \
-	$(X_EXTRA_LIBS)		\
-	$(XINERAMA_LIBS)	\
-	$(X_LIBS)		\
-	-lX11
+	$(XINERAMA_LIBS)
 
 # LIBGNOMEUI should be removed from gdmsetup at some point.
 # It is needed because it uses gnome_help_display_uri(), which
@@ -141,10 +138,7 @@
 	$(GDKPIXBUF_LIBS)	\
 	$(EXTRA_SETUP_LIBS)	\
 	$(LIBGNOMEUI_LIBS)	\
-	$(top_builddir)/vicious-extensions/libviciousui.a \
-	$(X_LIBS)		\
-	-lXau			\
-	-lX11
+	$(top_builddir)/vicious-extensions/libviciousui.a
 
 gdmphotosetup_LDADD = \
 	libgdmcommon.a		\
Index: gdm-2.18.1/gui/greeter/Makefile.am
===================================================================
--- gdm-2.18.1.orig/gui/greeter/Makefile.am	2007-04-09 07:15:10.000000000 +0200
+++ gdm-2.18.1/gui/greeter/Makefile.am	2007-05-02 11:07:23.000000000 +0200
@@ -87,10 +87,7 @@
 	$(GDK_LIBS)		\
 	$(GDKPIXBUF_LIBS)	\
 	$(GREETER_LIBS)		\
-	$(X_EXTRA_LIBS)		\
-	$(XINERAMA_LIBS)	\
-	$(X_LIBS)		\
-	-lX11
+	$(XINERAMA_LIBS)
 
 EXTRA_DIST = \
 	gdmthemetester.in \
